core/src/test/resources/ConfigurationUtils/stub4testtool/EsbJmsListener/original.xml

```diff
@@ -35,8 +35,8 @@
 			</pipe>
 
 			<pipe name="Read input CSV file"
-                  className="org.frankframework.pipes.FilePipe" actions="read"
-                  outputType="stream">
+                  className="org.frankframework.pipes.FixedResultPipe">
+				<param name="filename" sessionKey="filePathName"/>
```

Comment:
Is deze sessionkey hier ook nodig ?

Reply:
Ik denk het wel. Hoe weet de FixedResultPipe anders waar hij de filename parameter mee moet vullen?

Ik heb dit overigens niet als configuratie getest, dit wordt alleen door de `ConfigurationDigesterTest` gebruikt.

Reply:
Ik denk dat het beide kan, en zag eerder niet dat in de pipe hier voor de sessionkey gevuld werd.

Reply:
Ok!